google/cloud/forseti/services/cli.py

```diff
@@ -203,13 +203,108 @@ def define_inventory_parser(parent):
         help='Inventory id to get')
 
 
+def define_config_parser(parent):
+    """Define the config service parser.
+
+    Args:
+        parent (argparser): Parent parser to hook into.
+    """
+
+    service_parser = parent.add_parser('config', help='config service')
+
+    action_subparser = service_parser.add_subparsers(
+        title='action',
+        dest='action')
+
+    _ = action_subparser.add_parser(
+        'show',
+        help='Show the config')
+
+    delete_config_parser = action_subparser.add_parser(
+        'delete',
+        help='Deletes an item from the config')
+    delete_config_parser.add_argument(
+        'key',
+        type=str,
+        help='Key to delete from config')
+
+    set_config_parser = action_subparser.add_parser(
+        'set',
+        help='Model to switch to')
+    set_config_parser.add_argument(
+        'key',
+        type=str,
+        help='Key to set value for in config')
+    set_config_parser.add_argument(
+        'value',
+        type=str,
+        help='Value to set in config')
+
+
+def define_model_parser(parent):
+    """Define the model service parser.
+
+    Args:
+        parent (argparser): Parent parser to hook into.
+    """
+    service_parser = parent.add_parser('model', help='model service')
+    action_subparser = service_parser.add_subparsers(
+        title='action',
+        dest='action')
+
+    use_model_parser = action_subparser.add_parser(
+        'use',
+        help='Context switch into the model.')
+    use_model_parser.add_argument(
+        'model',
+        help='Model to switch to, either hash or name'
+        )
+
+    _ = action_subparser.add_parser(
+        'list',
+        help='List all available models')
+
+    delete_model_parser = action_subparser.add_parser(
+        'delete',
+        help='Deletes an entire model')
+    delete_model_parser.add_argument(
+        'model',
+        help='Model to delete')
+
+    create_model_parser = action_subparser.add_parser(
```

Comment:
Since model can be created here, shall we still keep the create_model in explainer?